main-actions/src/main/scala/sbt/Sync.scala

```diff
@@ -35,19 +37,22 @@ object Sync {
       store: CacheStore,
       inStyle: FileInfo.Style = FileInfo.lastModified,
       outStyle: FileInfo.Style = FileInfo.exists,
+      fileConverter: FileConverter
   ): Traversable[(File, File)] => Relation[File, File] =
-    sync(store, inStyle)
+    sync(store, inStyle, fileConverter)
 
+  /** this function ensures that the latest files in /src are also in /target, so that they are synchronised */
   def sync(
       store: CacheStore,
       inStyle: FileInfo.Style = FileInfo.lastModified,
+      fileConverter: FileConverter
   ): Traversable[(File, File)] => Relation[File, File] =
```

Comment:
This is a bincopat breakage so we'd need an overload of sync here that calls

```suggestion
  /** this function ensures that the latest files in /src are also in /target, so that they are synchronized */
  def sync(
      store: CacheStore,
      inStyle: FileInfo.Style = FileInfo.lastModified,
  ): Traversable[(File, File)] => Relation[File, File] =
    sync(store, inStyle, MappedFileConverter.empty)
    
  /** this function ensures that the latest files in /src are also in /target, so that they are synchronized */
  def sync(
      store: CacheStore,
      inStyle: FileInfo.Style = FileInfo.lastModified,
      fileConverter: FileConverter
  ): Traversable[(File, File)] => Relation[File, File] =
```